text/1029-rbac.rst

```diff
@@ -240,6 +242,23 @@ Requires ``cfg::perm::configure_allow_user_specified_id``
 The configuration will be done with annotations on Config objects,
 like everything else with the config system.
 
+Data branch restriction
+-----------------------
+
+Roles are also extended with a ``branches`` field, enumerating the branches
+that the role is allowed to connect to. The default value is ``{ '*' }``,
+signifying all branches.
```

Comment:
We don't allow creating a branch named `*`, right?

Reply:
You can use ` to escape any char sequence to be an identifier. There are a few exceptions, but I don't think `*` is one of them:

```
_localdev:main> create empty branch `*`;
OK: CREATE BRANCH
```

So we have a few options:
- this is allowed, if a user has a branch named `*` and adds `*` to a role's branches, the role has access to all branches. So they cannot restrict access to just `*`. It's a weird thing to do anyway.
- we forbid creating branches named `*`,
- we forbid all identifiers from containing `*`,

I would actually prefer first option.